pkgs/tools/audio/beets/default.nix

```
@@ -26,18 +27,34 @@ lib.makeExtensible (self: {
       rev = "v${version}";
       hash = "sha256-fT+rCJJQR7bdfAcmeFRaknmh4ZOP4RCx8MXpq7/D8tM=";
     };
+    extraPatches = [
+      # Bash completion fix for Nix
+      ./patches/bash-completion-always-print.patch
+      (fetchpatch {
+        # Fix unidecode>=1.3.5 compat
+        url = "https://github.com/beetbox/beets/commit/5ae1e0f3c8d3a450cb39f7933aa49bb78c2bc0d9.patch";
+        hash = "sha256-gqkrE+U1j3tt1qPRJufTGS/GftaSw/gweXunO/mCVG8=";
+      })
+      # Fix embedart with ImageMagick 7.1.1-12
+      # https://github.com/beetbox/beets/pull/4839
+      ./patches/fix-embedart-imagick-7.1.1-12.patch
```

Why are we not fetching the patch?

```suggestion
      ./fix-embedart-imagick-7.1.1-12.patch
```

---

Oh. I should add a comment that the patch I submitted upstream doesn't apply to the stable version, as the code has been refactored since. Should probably add that as a comment